packages/metadata-crawler/.gitignore

```diff
@@ -0,0 +1,3 @@
+lib/
```

Comment:
Did you mean to include this? Not a problem, just checking

Reply:
Yes, the metadata-crawler/lib was making me crazy. I know that it's not the way to do it, but this will avoid pushing things that shouldn't be in the repo